### PR TITLE
docs: document dual structure

### PR DESCRIPTION
**Pull request by gregatgoogle**
Relevant for https://github.com/unitedforukraine/signpost-template/issues/103

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,3 +148,18 @@ You need to generate a new Zendesk OAuth token for your site for Zendesk logging
...
    - Substitute <new_oauth_client_id>, <your_email> and <api_token>
    - Get an API token from https://signpost-global.zendesk.com/admin/apps-integrations/apis/zendesk-api/settings/tokens/
+
+## Architectural Decisions
+
+### Dual content structure
+
+There are two possible versions of content in a Zendesk instance. They are both
+explained at
+[Figma](https://www.figma.com/file/Gkxqy6IGhG1WYtEO6t9oOF/Default-template-%2F-Signpost?node-id=258%3A19009).
```
Comment (EOkoyomon) [inline README.md:158]: nit: maybe include the photos? Can also do this after.

Beporsed:
image

UFU:
image

**Review by EOkoyomon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gregatgoogle) [reply README.md:158]: Duplicate photos can more easily get out of date and add maintenance burden. A direct link should be enough.

Comment (EOkoyomon) [reply README.md:158]: Ack, but that assumes that everyone will have access to that figma. but ok, fine with me.

Comment (gregatgoogle) [reply README.md:158]: If they need it, they'll request access. If they need that information, they'll need that access anyway.

**Pull request merged**